packages/meteor-gazelle-core/lib/layout.html

```diff
@@ -0,0 +1,9 @@
+<template name="layout">
+  <div class="main-container">
+    {{> Template.dynamic template=header}}
+    <div class="main-content">
+      {{> Template.dynamic template=main}}
+    </div>
+    {{> Template.dynamic template=footer}}
+  </div>
+</template>
```

Comment:
Yeah, it's a flow router thing. It handles this differently than iron router. Now instead of doing the login check in the base template using a template helper, instead flow router will redirect to a different route which doesn't use this layout template but has its own.